src/makeRobot.js

```diff
@@ -39,6 +39,86 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(number) {
+      switch (true) {
+        case number === undefined:
+          this.coords.y += 1;
+          break;
+        case number >= 0:
+          this.coords.y += number;
+          break;
+        default:
+      }
```

Comment:
You don't need to use switch/case everywhere in this task. Just add a default value and make calculations. Fix it, everywhere, please  
```suggestion
    goForward(number = 1) {
      if (number >= 0) {
       ...
     }
```